crates/bevy_reflect/derive/src/impls/common.rs

```diff
@@ -156,3 +156,25 @@ pub fn common_partial_reflect_methods(
         #debug_fn
     }
 }
+
+pub fn reflect_auto_registration(meta: &ReflectMeta) -> Option<proc_macro2::TokenStream> {
```

Comment:
Any reason not to include this on opaque types?

Reply:
Didn't really think about that, will look into adding support for opaque types as well.

Reply:
Added automatic registration for opaque types